tests/topotests/bgp_evpn_overlay_index_gateway/test_bgp_evpn_overlay_index_gateway.py

```diff
@@ -128,12 +128,6 @@ def setup_module(mod):
         )
         return
 
-    if topotest.version_cmp(kernelv, "4.15") == 0:
```

Comment:
would this test start to fail if someone screws up the bind again?

Reply:
I don't know. Probably it will. But that's correct behavior so I'm not sure I get the question.

Reply:
We've had many iterations of the bind code.  We *Believe* we've hit on a version that works across many kernels and values set.  If anyone makes changes to it in the future I would want our test suites to catch this.  This sure seems like a bit of code that ensures that we are testing different values.